### PR TITLE
[animations] Add screenshots to pubspec.yaml

### PR DESCRIPTION
**Pull request by szakarias**
Adds screenshot to pubspec.yaml to be shown on pub.dev

### CODE REVIEW

File: packages/animations/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+## 2.0.6
+* Adds screenshots to pubspec.yaml
```
Comment (goderbauer) [inline packages/animations/CHANGELOG.md:2]: nit: blank line after this line

File: packages/animations/pubspec.yaml [hunk 2]
```diff
@@ -16,3 +16,15 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   vector_math: ^2.1.0
+
+screenshots:
+  - description: 'Thumbnail of a container transform animation.'
```
Comment (goderbauer) [inline packages/animations/pubspec.yaml:21]: I didn't see it in the example package, but do these descriptions show up somewhere on pub?

File: packages/animations/pubspec.yaml [hunk 3]
```diff
@@ -16,3 +16,15 @@ dev_dependencies:
...
+    path: example/demo_gifs/container_transform_thumbnail.webp
```
Comment (goderbauer) [inline packages/animations/pubspec.yaml:22]: I am not super in-love with how this thumbnail looks. But i also don't have a better idea of what to put here, so LGTM.

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/animations/CHANGELOG.md (hunk 1 above)
Comment (szakarias) [reply packages/animations/CHANGELOG.md:2]: Done.

File: packages/animations/pubspec.yaml (hunk 2 above)
Comment (szakarias) [reply packages/animations/pubspec.yaml:21]: No, they don't.

File: packages/animations/pubspec.yaml (hunk 3 above)
Comment (szakarias) [reply packages/animations/pubspec.yaml:22]: I can see what you mean. Hopefully it is good enough to encourage users to trigger the image carousel.

**Pull request merged**